sdk/core/azure-core-tracing-opentelemetry/pom.xml

```diff
@@ -35,19 +35,24 @@
     <dependency>
       <groupId>io.opentelemetry</groupId>
       <artifactId>opentelemetry-api</artifactId>
-      <version>0.2.4</version> <!-- {x-version-update;io.opentelemetry:opentelemetry-api;external_dependency} -->
+      <version>0.6.0</version> <!-- {x-version-update;io.opentelemetry:opentelemetry-api;external_dependency} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
       <version>1.7.0-beta.2</version> <!-- {x-version-update;com.azure:azure-core;current} -->
     </dependency>
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-core</artifactId>
+      <version>1.23.0</version> <!-- {x-version-update;io.opentelemetry:grpc-core;external_dependency} -->
```

Comment:
Looks like a copy/paste 
x-version-update;io.opentelemetry:grpc-core;external_dependency
should be
x-version-update;io.grpc:grpc-core;external_dependency